nservicebus/endpoint/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Endpoint
+summary: Describe general concept of Endpoint and Endpoint Instance.
+tags:
+- Endpoint
+- Endpoint Instance
+---
+
+A solution built on NServiceBus consists of a number of logical Endpoints that communicate via [Messaging](/nservicebus/messaging).
+
+## Endpoint
+
+An Endpoint is a design-time concept that has a name and a collection of associated Message Handlers and Sagas. A definition of an Endpoint lives as a group of artifacts in a source code repository. An Endpoint (the binaries that result from building the endpoint's source code) can be deployed to a number of machines and environments. Each such deployment is a separate Endpoint Instance.
```

Comment:
Is the part about artifacts in a source code repository adding any value? The fact that we then talk about an Endpoint as a compiled binary thing just adds to the confusion too. I wonder if we can't collapse both sections into something simpler.

>An Endpoint is a logical entity that communicates with other Endpoints via messaging. Each Endpoint has an identifying name and contains a collection of Message Handlers and Sagas. An Endpoint can be deployed to a number of machines and environments. Each such deployment is a separate Endpoint Instance. Each Endpoint Instance has an [Input Queue](/nservicebus/endpoint/specify-input-queue-name.md) which contains messages for the Endpoint Instance to process. 
>
>It is common for each Endpoint to have a single Endpoint Instance. As Endpoints need to [scale-out](/nservicebus/scalability-and-ha/scale-out.md), additional Endpoint Instances can be added. This collection of Endpoint Instances still represents a single logical Endpoint. 
>
>NOTE: In a scale-out scenario multiple Endpoint Instances can share a single Input Queue. 

Thoughts?